src/test/java/com/znsio/teswiz/screen/android/jiocinema/JioCinemaScreenAndroid.java

```diff
@@ -49,4 +52,12 @@ public boolean isMovieNumberVisibleOnScreen(int movieNumberOnScreen) {
                 Target.region(AppiumBy.xpath(String.format(visibleMovieNumberXpath, movieNumberOnScreen))));
         return isMovieVisibleOnScreen;
     }
+
+    @Override
+    public JioCinemaScreen swipeTrendingItem(String direction, int movieNumberOnScreen) {
+        LOGGER.info("Swiping right movie number : " + movieNumberOnScreen);
+        WebElement movieTrending = driver.waitTillElementIsVisible(By.xpath(String.format(movieXpath, movieNumberOnScreen)));
+        driver.horizontalSwipeWithGesturesPlugin(movieTrending,direction);
```

Comment:
rename method name to horizontalSwipeWithGesture

Reply:
Renamed to horizontalSwipeWithGesture